lib/display/display.ex

```diff
@@ -19,13 +19,18 @@ defmodule Inky.Display do
 
   @spec spec_for(:impression) :: Inky.Display.t()
   def spec_for(type = :impression) do
+    width = 600
+    height = 448
+
     %__MODULE__{
       type: type,
-      width: 600,
-      height: 448,
-      packed_resolution: <<2, 88, 1, 192>>, # I used the struct.pack in Python to generate this
+      width: width,
+      height: height,
+      packed_resolution:
+        <<width::unsigned-big-integer-size(16)>> <> <<height::unsigned-big-integer-size(16)>>,
+      packed_dimensions: packed_dimensions(type, width, height),
```

Comment:
I've added `packed_dimensions` here in an effort to get the display to show the buffer at full size instead of 1/4 the desired size. It's used in `lib/hal/impression/rpihal.ex` by the `do_update` function.